### ISSUE TITLE
Convert code to discretization to return \Delta A instead of A

### ISSUE THREAD

**Issue by jlperla** [econtoolkit/continuous_time_methods#19]
I think there are numerical instabilities being introducing for large numbers of grid points due to the division by $\Delta$ and $\Delta^2$ in the definitions.

I think I made all of the changes to the algebra document, though you should check very carefully in conjunction with #11 and then #13.   After the algebra is verified, and before you do $13, you should convert the discretization code to match the algebra in the document.

Basically, the discretize_univariate_diffusion.m file just needs to match the new algebra for the A to return, and the checks in the test suite are modified to the new A.

**Comment by jlperla** [comment]
Have you done this yet?  I think so.

**Comment by stevenzhangdx** [comment]
Yes, it is done.
